Helper/Api/Orders.php

```diff
@@ -1101,10 +1106,30 @@ private function getDiscountPayload(
     {
         foreach ($rates['data'] as $rate) {
             if ($rate['value'] == $taxPercent / 100) {
+                if ($order->getAppliedRuleIds()) {
+                    $ruleIdArray = explode(',', $order->getAppliedRuleIds());
+                }
+                $ruleName = '';
+                $ruleSku = '';
+
+                /**
+                 * Handle possibility of multiple discount rules being applied - combine all names in one string
+                 * and form sku from its combined name
+                 */
+                foreach ($ruleIdArray as $ruleId) {
+                    $rule = $this->ruleRepository->getById($ruleId);
+                    if ($rule->getRuleId()) {
+                        $ruleName .= $rule->getName() . ', ';
+                    }
+                }
+
+                $ruleName = rtrim($ruleName, ', ');
+                $ruleSku = $this->formatRuleSku($ruleName);
```

Comment:
can we push some of this data into `extra` in the line? It won't be used, but it would help the debug later on, if there any any bugs

like 

```
'extra' => [
   'magneto_discount_rules' => [
    ['id'=> , 'name'=> ]
   ]
]
```

Reply:
I did not mean the info hook extra, I mean the extra from order_item

see example: https://github.com/storekeeeper-shopapi/shop-examples/blob/c10a65a6e1d0174885a317f112124eb84679c624/webroot/orderAdvancedProductTypes.php#L66

![image](https://github.com/storekeeper-company/magento2-plugin/assets/1073778/08511db4-d00c-4361-8fcd-292e84468e19)

Reply:
can you add it to the discountpayload as well?